onnxruntime/core/providers/openvino/openvino_execution_provider.h

```diff
@@ -44,6 +65,19 @@ struct OpenVINOExecutionProviderInfo {
       #elif defined OPENVINO_CONFIG_VAD_F
       device_type_ = "HETERO:FPGA,CPU";
       precision_ = "FP32";
+      #elif defined OPENVINO_CONFIG_HETERO
+        #ifdef DEVICE_NAME
+          #define DEVICE DEVICE_NAME
+        #endif
+      dev_type = DEVICE;
+      if (dev_type.find("HETERO") == 0) {
```

Comment:
Indent the statements within this if block

Reply:
> Indent the statements within this if block

Done